gci/tests.py

```diff
@@ -1,3 +1,21 @@
-# from django.test import TestCase
 
-# Create your tests here.
+from django.test import TestCase
+from gci.models import Students
+
+# models test
+class StudentsModelTest(TestCase):
```

Comment:
Use `TransactionTestCase` as it lessens a lot of hassle with databases. And please use `setUp` and `tearDown`.

Reply:
https://docs.djangoproject.com/en/1.11/topics/testing/overview/